kin-recovery/recovery-sample/src/main/java/recovery/sample/BackupAndRestoreActivity.java

```diff
@@ -35,15 +38,16 @@ protected void onCreate(Bundle savedInstanceState) {
 		recoverAccount = findViewById(R.id.recover_account);
 		restoredAccountBalance = findViewById(R.id.balance_value);
 		restoredAccountPublicAddress = findViewById(R.id.public_address_value);
-//		balanceProgress = findViewById(R.id.balance_progress);
+		balanceProgress = findViewById(R.id.balance_value_progress);
+		publicAddressProgressBar = findViewById(R.id.public_address_value_progress);
 
 		createNewAccount.setOnClickListener(this);
 		backupCurrentAccount.setOnClickListener(this);
 		recoverAccount.setOnClickListener(this);
 
-		// TODO: 13/03/2019 should we support production?
-		backupAndRestorePresenter = new BackupAndRestorePresenterImpl(getBackupManager(),
-			getKinClient(NetWorkType.TEST));
+		KinClient kinClient = getKinClient(NetWorkType.TEST);
+		kinClient.clearAllAccounts();
```

Comment:
why we are clearing all the accounts?

Reply:
Though it will be better to have a clear sample each time the user want to use it.
Do you think we shouldn't clear them?